src/Sylius/Bundle/ShopBundle/Resources/views/Common/Form/_login.html.twig

```diff
@@ -1,6 +1,7 @@
 <div class="one field" id="sylius-api-login">
-    {{ form_errors(form.email) }}
-    {{ form_row(form.email, {'attr': {'data-url': path('sylius_shop_ajax_user_check_action')}}) }}
+    {{ form_errors(form.email, sylius_test_form_attribute('login-errors')) }}
+    {% set path = path('sylius_shop_ajax_user_check_action')  %}
+    {{ form_row(form.email, sylius_merge_recursively(sylius_test_form_attribute('login-email'), {'attr': {'data-url': path}})) }}
```

Comment:
Why wouldn't we go with a filter? There is `merge` filter in Twig, it would make sense to add `merge_recursive` filter also instead of a function - so this line would become `sylius_test_form_attribute('login-email')|sylius_merge_recursive({'attr': {'data-url': path}})` instead of `sylius_merge_recursively(sylius_test_form_attribute('login-email'), {'attr': {'data-url': path}})`.